packages/caip/src/caip2/caip2.ts

```diff
@@ -126,20 +138,16 @@ export const fromCAIP2: FromCAIP2 = (caip2) => {
       const chain = ChainTypes.Cosmos
```

Comment:
Stylistic pref to just have ChainTypes.Cosmos inline inside switch now that there are two types for the same namespace.